src/braket/pennylane_plugin/translation.py

```diff
@@ -60,16 +76,174 @@ def supported_operations() -> FrozenSet[str]:
     return frozenset(_OPERATION_MAP.keys())
```

Comment:
Instead of using the dict, you can use `translate_operation.registry` to get a list of implementations. This way there's only one source of truth.

Reply:
good call